lib/webview/src/component/DiffView.tsx

```diff
@@ -318,9 +319,15 @@ function toPrismHighlightOptions(
       return DEFAULT_PRISM_OPTIONS;
 
     default:
-      console.warn(
-        `Can't find Prism grammar for language ${languageId}, use default one`
-      );
+      sendMessage({
+        type: "showError",
+        error: {
+          title: `Unable to highlight syntax for language ${languageId}`,
+          message: `We could not find a matching Prism grammar for language ${languageId}. We used the default one (${DEFAULT_PRISM_OPTIONS.language}). Please [open an issue](https://github.com/rubberduck-ai/rubberduck-vscode/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Add%20syntax%20highlight%20for%20language%20%22${languageId}%22) to ask for supporting this language.`,
```

Comment:
This would create a link that works to create a new GitHub issue, like this: https://github.com/rubberduck-ai/rubberduck-vscode/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Add%20syntax%20highlight%20for%20language%20%22typescript%22

IMO, this is useful for error messages that we create. It will simplify reporting errors since we could also include content in the body (eg. `&body=${encodeURIComponent(error)}`)